Build-and-Deploy-Your-First-Dashboard.mdx

```diff
@@ -0,0 +1,420 @@
+
+    </Tip>
+
+  </Accordion>
+
+{" "}
+<Accordion title="2. Prepare data">
+Click on the **+ aggregations** , this will create ```SQL SELECT * FROM "aggregations" ```   
+in the SQL editor.
+
+Click on **Preview data** to run the SQL and **Next** once you see the preview data.
+     <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/query-builder.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/query-builder-dark.png"
+  alt="Hero Dark"
+/>
+    </Frame>
+
+
+ <Tip>
+    If you are more advanced with SQL, you can add more tables and customise your SQL.
+    </Tip>
+  </Accordion>
+
+{" "}
+<Accordion title="3. Build chart">
+  
+  For your first chart select a **Grouped bar**, and choose **country** as the x-axis and  **demand** as the y-axis.
+  the corresponding x- & y-axis
+  
+  For future charts see [Chart Configuration](/charts/create-chart) for all chart settings. 
+  
+  
+
+     <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/chart-builder-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/chart-builder-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+</Accordion>
+
+  <Accordion title="4. Add chart filters">
+  To add chart filters 
+    You can set up filters by clicking on **configure filters** and selecting the relevant column and filter type.
+
+
+    <Steps>
+      <Step title="Select Table ">
+      **1. Raw backing tables:** filter any column in the raw data that is powering this specific chart
+
+**2. Computed table:** filter any column from the data table that was computed by the SQL written for this chart
+
+**3. Variables in the chart SQL:** inject variables into the SQL code that is backing this chart
+
+      </Step>
+
+      
+
+      <Step title="Configure Filter ">
+
+
+**1. Table:** select the table to filter or "Query variables"
+
+**2. Column:** select the column or variable to filter on
+
+**3. Filter:** configure the filter as needed. Options will be provided based on the type of filter selected
+
+      </Step>
+    
+
+      
+      </Steps>
+    ![configure-filters](/images/filters/filters-guide-4.png)
+
+
+
+  
+    <Tip>
+    For a more comprehensive overview, check out our [in-depth filtering guide](/filters/filters-in-depth).
+    </Tip>
+    
+  </Accordion>
+
+    <Accordion title="5. Save chart">
+
+  Click **Save and add to dashboard** when you are happy with your chart, this will save the chart. You could also
+  create a copy of the chart by clicking **Duplicate**.
+
+  </Accordion>
+</AccordionGroup>
+
+### Configure your dashboard
+
+You are able to customise your dashboard by adding dashboard filters, controlled filters and additional charts. 
+
+To resize charts you can simply drag the chart to the size you would like.
```

Comment:
let's mention they need to hover over the bottom right corner of a chart, i saw people didn't realize that before